classes/autoptimizeCriticalCSSCore.php

```diff
@@ -53,6 +53,9 @@ public function run() {
 
             // Extend conditional tags on plugin initalization.
             add_action( apply_filters( 'autoptimize_filter_ccss_extend_types_hook', 'init' ), array( $this, 'ao_ccss_extend_types' ) );
+
+            // When autoptimize cache is cleared, also clear transient cache for page templates
+            add_action( 'autoptimize_action_cachepurged', array('autoptimizeCriticalCSSCore', 'ao_ccss_clear_page_tpl_cache'), 10, 0 );
```

Comment:
great idea!

Reply:
Cool.

I have changed
```array('autoptimizeCriticalCSSCore', 'ao_ccss_clear_page_tpl_cache')```
to
```array( 'autoptimizeCriticalCSSCore', 'ao_ccss_clear_page_tpl_cache' )```
here too